share/jupyter/voila/templates/materialscloud/spinner.macro.html.j2

```diff
@@ -16,6 +16,6 @@
   <div id="loading">
     <img style="width: 150px; text-align: center;" src="{{ base_url }}voila/static/images/mcloud_spinner.svg"/>
     <h2 id="loading_text">Running {{nb_title}}...</h2>
-    This page should redirect you. If it doesn't, <a href="{{ open_url }}">click here</a>.
+    Now Loading. If it doesn't finish loading, <a href="{{ open_url }}">click here</a>.
```

Comment:
```suggestion
    Loading... If this takes longer than ~5 seconds, <a href="{{ open_url }}">click here</a>.
```

I think this may be better. The loading _should_ be done within 5 seconds.

Reply:
@CasperWA I have updated the text as per your review.